<?xml version="1.0"?>
<launch>
    <arg name="target" default="[10, 5, 0]" />
    <arg name="radius" default="5" />
    <arg name="polygon" default="6" />
    <arg name="is_ccw" default="true" />
    <arg name="is_relative" default="false" />
    <arg name="isgazebo" default="false" />
    <arg name="iswamv" default="false" />
    <arg name="isfake" default="false" />
    <arg name="center_x" default="10" />
    <arg name="center_y" default="10" />

    <param name="/use_sim_time" value="$(arg isgazebo)"/>

    <node pkg="robotx_nav" type="move_base_loiter.py" name="loiter_behavior" >
        <remap from="/odom" to="/odometry/filtered/global" if="$(arg isgazebo)" />
        <remap from="/odom" to="/odometry/filtered/global" if="$(arg iswamv)" />
        <param name="radius" value="$(arg radius)" type="double"/>
        <param name="polygon" value="$(arg polygon)" type="int" />
        <param name="is_ccw" value="$(arg is_ccw)" type="bool"/>
	<param name="center/x" value="$(arg center_x)" type="double"/>
	<param name="center/y" value="$(arg center_y)" type="double"/>
        <param name="is_relative" value="$(arg is_relative)" />
    </node>

</launch>
